stats.py

```diff
@@ -201,6 +201,22 @@ def execute(self):
 # Cmdline parsing definitions
 # #
 
+def _callback(option, opt_str, value, parser):
```

Comment:
this doesn't accept pipe anymore but make parsing a list of arguments easier. Is there a better way ?